### PR TITLE
[SNMP] Update extension-to-physical-entity-mib.md to add a description about the Entity sensor MIB extension

### PR DESCRIPTION
**Pull request by keboliu**
Extend the Entity sensor MIB to include more sensors.
currently, Entity sensor MIB only include the SFP DOM sensors, in this extension, will add below sensors:
1. the PSU voltage, current, power, temp
2. Chassis fan speed, PSU fan speed
3. Thermal sensors avalaible on the switch(ASIC, CPU, Ambt, etc.)

### CODE REVIEW

File: doc/snmp/extension-to-physical-entity-mib.md [hunk 1]
```diff
@@ -372,3 +375,68 @@ New test cases will be added to cover the new MIB entries:
...
 4. Remove/Add DB entries from related tables to see whether MIB info can be correctly updated.
 5. Currently, each platform API is tested by sonic-mgmt, see [here](https://github.com/Azure/sonic-mgmt/tree/master/tests/platform_tests/api). We will add regression test case for each newly added platform API to verify them.
+
+
+## 2. Entity Sensor MIB Extension
+
+### 2.1 Entity Sensor MIB Extension Scope
+
+Currently SONiC already have a RFC3433 Entity sensor MIB implementation,  Transceiver embeded sensors like temperature, voltage, rx_power, tx_power and tx_bias are alreay included. After Entity MIB was extended as desribed in Section 1, Entity MIB will also be extened to cover more sensors. 
+Thermal sensors besides the Transceiver embeded thermal sensors which avalaible in the STATE_DB will all be added to the Entity Sensor MIB support.
```
Comment (jleveque) [inline doc/snmp/extension-to-physical-entity-mib.md:385]: Typo: s/embeded/embedded/

File: doc/snmp/extension-to-physical-entity-mib.md [hunk 2]
```diff
@@ -372,3 +375,68 @@ New test cases will be added to cover the new MIB entries:
...
+PSU temperature, voltage, power and current will added, FAN tachometer will be added as well.
+
+### 2.1 Entity Sensor MIB Data Source
+
+All new added MIB will be initilized and updated based on the data stored in the STATE_DB, to be more specific:
```
Comment (jleveque) [inline doc/snmp/extension-to-physical-entity-mib.md:390]: Typo: s/initilized/initialized/

**Review by jleveque — CHANGES_REQUESTED** [review]

**Comment by jleveque** [comment]
Looks good to me. @qiluo-msft to review from an SNMP perspective.

**Comment by keboliu** [comment]
Hi, @jleveque would you please check my update regarding your comments?

**Review by jleveque — APPROVED** [review]

**Pull request merged**